### ISSUE TITLE
"Insert recent" not updated 

### ISSUE THREAD

**Issue by ha24-1** [dirkwhoffmann/virtualc64#816]
Hi,
another small issue in Version 5.0.1 (240724): the "Insert recent"-list is not updated if .d64 will be inserted via "Insert...".
Drag&Drop of .d64-Files into VC64 is updating the "Insert recent"-list.

Best regards!
H.

**Comment by dirkwhoffmann** [comment]
This one was easy.

The code snippet

```Swift
                    try self.mm.addMedia(url: url,
                                             allowedTypes: [ .D64, .T64, .PRG, .P00, .G64 ],
                                             drive: id,
                                             options: [.force])

                    } catch {
                        self.showAlert(.cantInsert, error: error, async: true)
                    }
```

had to be replaced by:

```Swift
                    try self.mm.addMedia(url: url,
                                             allowedTypes: [ .D64, .T64, .PRG, .P00, .G64 ],
                                             drive: id,
                                             options: [.force, .remember])

                    } catch {
                        self.showAlert(.cantInsert, error: error, async: true)
                    }
```

Thanks for reporting!

**Comment by dirkwhoffmann** [comment]
Fixed in v5.1
